### PR TITLE
PIM-6933: fix menu display in case of acl restriction

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

Fix render of menu item displaying even if they don't have sub items. Before this fix, if no subsection were visible by the user, the menu was crashing. Now, You need to be able to see at least one section to see the menu item.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | NA
| Added Behats                      | Todo
| Added integration tests           | NA
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Comment by LaureBro** [comment]
Hello, can you add the PIM ref in the changelog please?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/menu/tab.js [hunk 1]
```diff
@@ -63,7 +63,13 @@ define(
              * {@inheritdoc}
              */
             render: function () {
-                this.$el.empty().append(this.template({
+                this.$el.empty()
+
+                if (!this.config.to && 0 === this.items.length) {
```
Comment (tamarasaurus) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/menu/tab.js]: can you use !this.hasChildren() here ?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/menu/tab.js [hunk 2]
```diff
@@ -63,7 +63,13 @@ define(
              * {@inheritdoc}
              */
             render: function () {
-                this.$el.empty().append(this.template({
+                this.$el.empty()
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/menu/tab.js]: missing semicolon?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/menu/tab.js [hunk 3]
```diff
@@ -154,6 +160,15 @@ define(
                 if (event.target === this.code) {
                     this.items.push(event);
                 }
+            },
+
+            /**
+             * Does this tab has children elements
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/menu/tab.js]: have

**Pull request merged**